### ISSUE TITLE
Why has the "--learning_rate=" setting disappeared from Kohya LoRA Dreambooth?

### ISSUE THREAD

**Issue by noriwaru** [Linaqruf/kohya-trainer#49]
Perhaps unet_lr=--learning_rate"?
I am ignorant.

**Comment by soyusora** [comment]
I don't understand it either, can someone explain? how to set learning rate? I need to make it the same with unet lr

**Comment by Linaqruf** [comment]
If you set `--learning_rate = 1e-4` means both unet and text encoder trained at 1e-4, then you define `--text_encoder_lr = 5e-5`, means unet trained at 1e-4 (default learning rate) and text encoder trained at 5e-5, or you define `--unet_lr = 5e-5` means unet trained at 5e-5 and text_encoder trained at 1e-4 (default learning rate)

In conclusion i find it useless to set `--learning_rate` if you also set `--unet_lr` and `--text_encoder_lr`, because it will override the `--learning_rate` value.

But if it makes confusion i can re-add it in the next update.

![image](https://user-images.githubusercontent.com/50163983/216520814-be21f35e-6a7e-4ca6-a18a-9646f67f8654.png)
![image](https://user-images.githubusercontent.com/50163983/216520883-27c4b3d2-cd72-414f-b4e3-ceb50d436225.png)

**Comment by noriwaru** [comment]
That makes sense. Somehow, I knew that was the case. I think other people will probably understand, so I think it's okay to leave it as it is. thank you very much.
